tajo-catalog/tajo-catalog-client/src/main/java/org/apache/tajo/catalog/AbstractCatalogClient.java

```diff
@@ -805,11 +851,19 @@ public final boolean containFunction(final String signature, FunctionType funcTy
   }
 
   @Override
-  public final boolean alterTable(final AlterTableDesc desc) {
+  public final void alterTable(final AlterTableDesc desc) throws UndefinedTableException, DuplicateColumnException,
+      InsufficientPrivilegeException, DuplicateTableException {
 
     try {
       final BlockingInterface stub = getStub();
-      return isSuccess(stub.alterTable(null, desc.getProto()));
+      final ReturnState state = stub.alterTable(null, desc.getProto());
```

Comment:
CatalogServer.alterTable() throws only ```UndefinedTableException```. Did you add handling other exceptions for future work? Even though you did, I think that ```DuplicateTableException``` does not occur.